packages/loader/container-loader/src/container.ts

```diff
@@ -878,13 +878,17 @@ export class Container
 			try {
 				// Raise event first, to ensure we capture _lifecycleState before transition.
 				// This gives us a chance to know what errors happened on open vs. on fully loaded container.
-				this.mc.logger.sendTelemetryEvent(
-					{
-						eventName: "ContainerClose",
-						category: error === undefined ? "generic" : "error",
-					},
-					error,
-				);
+				// Log generic events instead of error events if container is in loading state, as most errors are not really FF errors
+				// and this can pollute telemetry for real errors
+				if (error !== undefined) {
```

Comment:
I do not think this check is correct, as we do want to see Closed event for normal close (without error).
I think the best course of action - always log, but make it an error category only if we have an error, and this._lifecycleState !== "loading"